apps/centralised-indexer/testing/containers/DynamoDbContainer.ts

```diff
@@ -0,0 +1,134 @@
+import Dockerode, { ContainerInfo, DockerOptions } from 'dockerode'
+import { DockerContainer } from '@defichain/testcontainers'
```

Comment:
Instead of exposing`DockerContainer` from `@defichain/testcontiners` , would moving this into `@defichain/testcontainer` instead be a better call?

Reply:
Hmm don't think DynamoDB container should be an offering of the core package, since it's for a single specific app's use-case